api-tests/specs/assign-roles/assign-roles.js

```diff
@@ -0,0 +1,31 @@
+const requests = require('../../data/assign-roles/assign-roles')
+const conf = require('../../config/conf')
+const sitesEndpointUri = ':8083/sites';
```

Comment:
Can you get these service URLs from the config server? These are the local configurations I think, won't work in Azure maybe?

Reply:
I might need help with this @alexbaxter